kubeless-ui.jsonnet

```diff
@@ -33,7 +33,9 @@ local host = "kubeless-ui";
           serviceAccountName: $.serviceAccount.metadata.name,
           containers_+: {
             default: kube.Container("ui") {
-              image: "bitnami/kubeless-ui:v0.2.3",
+              // FIXME: point to real tag
+              image: "bitnami/kubeless-ui:latest",
+              imagePullPolicy: "Always",
```

Comment:
should we wait for the new release `v0.2.4` ?

Reply:
I want to get this in front of people so they can test everything works, I created #15 to revisit this as soon as we have some validation.